Automatic tests stopped working with the release of PyMongo 4.0

```
tests/adapter/mongo/test_base.py::test_collection_stats ERROR            [  0%]

==================================== ERRORS ====================================
___________________ ERROR at setup of test_collection_stats ____________________

request = <SubRequest 'real_adapter' for <Function test_collection_stats>>
real_pymongo_client = MongoClient(host=['localhost:27017'], document_class=dict, tz_aware=False, connect=True)

    @pytest.fixture(scope="function")
    def real_adapter(request, real_pymongo_client):
        """Get an adapter connected to mongo database"""
        mongo_client = real_pymongo_client
    
        LOG.info("Connecting to database %s", REAL_DATABASE)
    
        database = mongo_client[REAL_DATABASE]
>       mongo_adapter = PymongoAdapter(database)

tests/conftest.py:682: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
scout/adapter/mongo/base.py:85: in __init__
    if database:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = Database(MongoClient(host=['localhost:27017'], document_class=dict, tz_aware=False, connect=True), 'realtestdb')

    def __bool__(self):
>       raise NotImplementedError("Database objects do not implement truth "
                                  "value testing or bool(). Please compare "
                                  "with None instead: database is not None")
E       NotImplementedError: Database objects do not implement truth value testing or bool(). Please compare with None instead: database is not None
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Added
 ### Changed
 ### Fixed
+- Freeze PyMongo lib to version<4.0 to keep supporting previous MongoDB versions
 
 
 ## [4.42]

--- a/requirements.txt
+++ b/requirements.txt
@@ -16,7 +16,7 @@ query_phenomizer
 Flask-Babel
 livereload
 python-dateutil
-pymongo>=3.7
+pymongo>=3.7,<4.0
 pathlib
 WeasyPrint==0.42.3
 xlsxwriter

